### PR TITLE
Create AIP for order votes

### PR DESCRIPTION
**Pull request by vusirikala**
Adding a new AIP for order votes

### CODE REVIEW

File: aips/aip-order-votes.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	if |pending_order_votes[commit_id]| >= 2f+1 {
+	    // Add order_vote_msg.qc to the block store, if not already present
+		// Send the block with commit_id to execution
+		// Update highest_ordered_cert
+		finalize(order_vote.ledger_info)
+	}
+}
+```
+
+### Proof of Safety
```
Comment (danielxiangzl) [inline aips/aip-order-votes.md:137]: I think it is better to remove proofs in this AIP. They are not rigorous enough.

**Review by zekun000 — APPROVED** [review]

File: aips/aip-order-votes.md (hunk 1 above)
Comment (vusirikala) [reply aips/aip-order-votes.md:137]: @zekun000 I see you approved already. Is it okay if I remove the proofs?

Comment (zekun000) [reply aips/aip-order-votes.md:137]: yeah I don't think it's necessary to have proofs here

Comment (vusirikala) [reply aips/aip-order-votes.md]: Done.

**Review by thepomeranian — APPROVED** [review]

**Pull request merged**